Fix compatibility with Cython 3.0.0
Remove unused import which seems to be removed in Cython 3.
https://github.com/wxWidgets/Phoenix/issues/2439
https://github.com/wxWidgets/Phoenix/pull/2441
https://github.com/wxWidgets/Phoenix/issues/2514
https://github.com/wxWidgets/Phoenix/commit/aeb557d01e7cd37176ebbf0f1ae6d0b53c115378
--- wx/svg/_nanosvg.pyx.orig
+++ wx/svg/_nanosvg.pyx
@@ -42,7 +42,6 @@ for manipulating the SVG shape info in memory.
 
 import sys
 
-cimport cython.object
 from cpython.buffer cimport (
     Py_buffer, PyObject_CheckBuffer, PyObject_GetBuffer, PyBUF_SIMPLE,
     PyBuffer_Release)
